setup.py

```diff
@@ -24,10 +24,11 @@
             'pyosf', 'requests[security]',
             'xlrd', 'openpyxl',
             'pyyaml', 'gevent', 'msgpack-python', 'psutil', 'tables'
+            'opencv-python',
             ]
 # some optional dependencies
 if platform == 'darwin':
-    required.append('pyobjc-core')
+    required.append('pyobjc-core', 'pyobjc-framework-Quartz')
```

Comment:
This raises a TypeError. I suggest `required.extend(['pyobjc-core', 'pyobjc-framework-Quartz'])`